scripts/setup-chains

```diff
@@ -70,8 +70,8 @@ CHAIN_1_RPC_ADDR="localhost:$CHAIN_1_RPC_PORT"
 CHAIN_0_SAMOLEANS=100000000000
 CHAIN_1_SAMOLEANS=100000000000
 
-"$ONE_CHAIN" gaiad "$CHAIN_0_ID" ./data $CHAIN_0_RPC_PORT 26656 6060 9090 $CHAIN_0_SAMOLEANS
-"$ONE_CHAIN" gaiad "$CHAIN_1_ID" ./data $CHAIN_1_RPC_PORT 26556 6061 9091 $CHAIN_1_SAMOLEANS
+"$ONE_CHAIN" gaiad "$CHAIN_0_ID" ./data $CHAIN_0_RPC_PORT 26656 6060 9090 9091 $CHAIN_0_SAMOLEANS
```

Comment:
If the `gaiad` invocation changed, did the branch/release also change? If so, maybe we should update docs/code to reflect the new release.

Reply:
Good point, l just undid those changes. I was working with a more recent gaia off a recent cosmos-sdk. We don't need these changes here. I think we need to update the docker images to use `gaia v4.0.4` and make the changes there.

Reply:
Opened https://github.com/informalsystems/ibc-rs/issues/702 for the gaia upgrade in CI